### ISSUE TITLE
`$ yarn check-all` command modifies some files  

### ISSUE THREAD

**Issue by sukjae** [reactjs/ko.reactjs.org#74]
Hi. 

I followed instruction and had test as mentioned. 

> Test the change
> If possible, test any visual changes in all latest versions of common browsers, on both desktop and mobile.
> Run **yarn check-all** from the project root. (This will run Prettier, ESLint, and Flow.)

but, after i run `yarn check-all`, it automatically modifies two files and result to fail for textlint while committing it.

here is my logs...

```
➜  ko.reactjs.org git:(master) git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean`
```

```
➜  ko.reactjs.org git:(master) git checkout -b translate-uncontrolled-components
Switched to a new branch 'translate-uncontrolled-components'`
```

```
➜  ko.reactjs.org git:(translate-uncontrolled-components) yarn check-all
yarn run v1.7.0
$ npm-run-all prettier generate-ids --parallel lint flow
$ yarn format:source && yarn format:examples
$ prettier --config .prettierrc --write "{gatsby-*.js,{flow-typed,plugins,src}/**/*.js}"
flow-typed/gatsby.js 400ms
flow-typed/glamor.js 19ms
flow-typed/hex2rgba.js 14ms
flow-typed/react-helmet.js 10ms
flow-typed/slugify.js 8ms
gatsby-browser.js 34ms
gatsby-config.js 77ms
gatsby-node.js 15ms
plugins/gatsby-remark-header-custom-ids/gatsby-client.js 27ms
plugins/gatsby-remark-header-custom-ids/gatsby-ssr.js 34ms
plugins/gatsby-remark-header-custom-ids/index.js 33ms
plugins/gatsby-remark-use-jsx/index.js 17ms
plugins/gatsby-source-react-error-codes/gatsby-node.js 17ms
plugins/gatsby-transformer-authors-yaml/gatsby-node.js 18ms
plugins/gatsby-transformer-home-example-code/gatsby-node.js 17ms
plugins/gatsby-transformer-versions-yaml/create-redirects.js 29ms
plugins/gatsby-transformer-versions-yaml/gatsby-node.js 14ms
src/components/ButtonLink/ButtonLink.js 22ms
src/components/ButtonLink/index.js 8ms
src/components/CodeEditor/CodeEditor.js 76ms
src/components/CodeEditor/index.js 12ms
src/components/CodeExample/CodeExample.js 21ms
src/components/CodeExample/index.js 9ms
src/components/Container/Container.js 12ms
src/components/Container/index.js 6ms
src/components/ErrorDecoder/ErrorDecoder.js 34ms
src/components/ErrorDecoder/index.js 6ms
src/components/Flex/Flex.js 13ms
src/components/Flex/index.js 8ms
src/components/Header/Header.js 8ms
src/components/Header/index.js 7ms
src/components/Layout/index.js 6ms
src/components/Layout/Layout.js 14ms
src/components/LayoutFooter/ExternalFooterLink.js 10ms
src/components/LayoutFooter/Footer.js 45ms
src/components/LayoutFooter/FooterLink.js 13ms
src/components/LayoutFooter/FooterNav.js 13ms
src/components/LayoutFooter/index.js 9ms
src/components/LayoutHeader/DocSearch.js 34ms
src/components/LayoutHeader/Header.js 51ms
src/components/LayoutHeader/HeaderLink.js 19ms
src/components/LayoutHeader/index.js 7ms
src/components/LayoutHeader/SearchSvg.js 8ms
src/components/MarkdownHeader/index.js 6ms
src/components/MarkdownHeader/MarkdownHeader.js 9ms
src/components/MarkdownPage/index.js 6ms
src/components/MarkdownPage/MarkdownPage.js 31ms
src/components/StickyResponsiveSidebar/index.js 6ms
src/components/StickyResponsiveSidebar/StickyResponsiveSidebar.js 39ms
src/components/TitleAndMetaTags/index.js 7ms
src/components/TitleAndMetaTags/TitleAndMetaTags.js 9ms
src/html.js 12ms
src/pages/404.js 13ms
src/pages/acknowledgements.html.js 24ms
src/pages/blog/all.html.js 46ms
src/pages/docs/error-decoder.html.js 21ms
src/pages/index.js 58ms
src/pages/jsx-compiler.html.js 15ms
src/pages/languages.js 28ms
src/pages/versions.js 18ms
src/prism-styles.js 29ms
src/site-constants.js 8ms
src/templates/blog.js 16ms
src/templates/codepen-example.js 20ms
src/templates/community.js 14ms
src/templates/components/ChevronSvg/index.js 11ms
src/templates/components/ExternalLinkSvg/index.js 8ms
src/templates/components/MetaTitle/index.js 22ms
src/templates/components/NavigationFooter/index.js 9ms
src/templates/components/NavigationFooter/NavigationFooter.js 31ms
src/templates/components/Sidebar/index.js 9ms
src/templates/components/Sidebar/ScrollSyncSection.js 28ms
src/templates/components/Sidebar/Section.js 20ms
src/templates/components/Sidebar/Sidebar.js 13ms
src/templates/docs.js 12ms
src/templates/tutorial.js 10ms
src/theme.js 53ms
src/types.js 9ms
src/utils/createLink.js 18ms
src/utils/createOgUrl.js 9ms
src/utils/findSectionForPath.js 10ms
src/utils/isItemActive.js 9ms
src/utils/loadScript.js 7ms
src/utils/patchDOMForGoogleTranslate.js 10ms
src/utils/sectionList.js 8ms
src/utils/slugify.js 7ms
src/utils/toCommaSeparatedList.js 12ms
$ prettier --config examples/.prettierrc --write "examples/**/*.js"
examples/16-3-release-blog-post/context-example.js 392ms
examples/16-3-release-blog-post/create-ref-example.js 17ms
examples/16-3-release-blog-post/fancy-button-example.js 15ms
examples/16-3-release-blog-post/forward-ref-example.js 30ms
examples/16-3-release-blog-post/hoc-theme-example.js 10ms
examples/16-4-release-blog-post/pointer-events-example.js 52ms
examples/components-and-props/composing-components.js 12ms
examples/components-and-props/extracting-components-continued.js 29ms
examples/components-and-props/extracting-components.js 36ms
examples/components-and-props/rendering-a-component.js 9ms
examples/context/motivation-problem.js 10ms
examples/context/motivation-solution.js 11ms
examples/context/multiple-contexts.js 17ms
examples/context/reference-caveats-problem.js 8ms
examples/context/reference-caveats-solution.js 11ms
examples/context/theme-detailed-app.js 22ms
examples/context/theme-detailed-theme-context.js 12ms
examples/context/theme-detailed-themed-button.js 10ms
examples/context/updating-nested-context-app.js 14ms
examples/context/updating-nested-context-context.js 6ms
examples/context/updating-nested-context-theme-toggler-button.js 8ms
examples/es5-syntax-example.js 8ms
examples/forwarding-refs/customized-display-name.js 11ms
examples/forwarding-refs/fancy-button-ref.js 7ms
examples/forwarding-refs/fancy-button-simple-ref.js 8ms
examples/forwarding-refs/fancy-button-simple.js 6ms
examples/forwarding-refs/fancy-button.js 6ms
examples/forwarding-refs/log-props-after.js 11ms
examples/forwarding-refs/log-props-before.js 8ms
examples/forwarding-refs/wrapped-component-with-function-name.js 6ms
examples/forwarding-refs/wrapped-component.js 6ms
examples/hello-world.js 6ms
examples/introducing-jsx.js 9ms
examples/jsx-simple-example.js 6ms
examples/react-component-reference/get-snapshot-before-update.js 11ms
examples/reconciliation/index-used-as-key.js 25ms
examples/reconciliation/no-index-used-as-key.js 27ms
examples/reference-react-forward-ref.js 11ms
examples/rendering-elements/render-an-element.js 7ms
examples/rendering-elements/update-rendered-element.js 10ms
examples/strict-mode/enabling-strict-mode.js 12ms
examples/strict-mode/side-effects-in-constructor.js 7ms
examples/tutorial-expanded-version.js 7ms
examples/uncontrolled-components/input-type-file.js 13ms
examples/update-on-async-rendering/adding-event-listeners-after.js 12ms
examples/update-on-async-rendering/adding-event-listeners-before.js 9ms
examples/update-on-async-rendering/adding-event-listeners-create-subscription.js 12ms
examples/update-on-async-rendering/definition-getderivedstatefromprops.js 7ms
examples/update-on-async-rendering/definition-getsnapshotbeforeupdate.js 7ms
examples/update-on-async-rendering/fetching-external-data-after.js 11ms
examples/update-on-async-rendering/fetching-external-data-before.js 10ms
examples/update-on-async-rendering/initializing-state-after.js 6ms
examples/update-on-async-rendering/initializing-state-before.js 9ms
examples/update-on-async-rendering/invoking-external-callbacks-after.js 6ms
examples/update-on-async-rendering/invoking-external-callbacks-before.js 14ms
examples/update-on-async-rendering/react-dom-properties-before-update-after.js 9ms
examples/update-on-async-rendering/react-dom-properties-before-update-before.js 10ms
examples/update-on-async-rendering/side-effects-when-props-change-after.js 7ms
examples/update-on-async-rendering/side-effects-when-props-change-before.js 6ms
examples/update-on-async-rendering/updating-external-data-when-props-change-after.js 12ms
examples/update-on-async-rendering/updating-external-data-when-props-change-before.js 12ms
examples/update-on-async-rendering/updating-state-from-props-after.js 6ms
examples/update-on-async-rendering/updating-state-from-props-before.js 6ms
examples/update-on-async-rendering/using-react-lifecycles-compat.js 8ms
$ node scripts/generateHeadingIDs.js content
$ flow
$ eslint .
Launching Flow server for /Users/sukjaelee/open-source-project/ko.reactjs.org
Spawned flow server (pid=91711)
Logs will go to /private/tmp/flow/zSUserszSsukjaeleezSopen-source-projectzSko.reactjs.org.log
No errors!
✨  Done in 10.61s.
```

```
➜  ko.reactjs.org git:(translate-uncontrolled-components) ✗ git status
On branch translate-uncontrolled-components
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   content/community/conferences.md
	modified:   content/docs/hooks-faq.md

no changes added to commit (use "git add" and/or "git commit -a")
```

```
➜  ko.reactjs.org git:(translate-uncontrolled-components) ✗ git add -A
➜  ko.reactjs.org git:(translate-uncontrolled-components) ✗ git commit -m "Translated docs/uncontrolled-components page"
husky > pre-commit (node v8.11.1)
  ↓ Stashing changes... [skipped]
    → No partially staged files found...
  ❯ Running linters...
    ❯ Running tasks for *.md
      ✖ textlint --rulesdir textlint/rules



✖ textlint --rulesdir textlint/rules found some errors. Please fix them and try committing again.

/Users/sukjaelee/open-source-project/ko.reactjs.org/content/docs/hooks-faq.md
 64:80   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
121:54   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
140:208  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
189:38   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
220:44   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
240:171  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
254:270  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
263:184  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
283:143  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
297:150  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
325:92   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
341:73   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
376:170  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
416:95   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
439:173  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
457:271  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
482:100  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
536:268  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
550:96   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
572:190  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
591:174  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
611:94   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
631:96   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
635:256  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
659:174  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
676:357  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
697:227  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
723:83   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
737:152  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
749:77   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
772:76   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
782:89   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
794:180  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
804:147  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
833:82   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
841:178  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
858:163  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
875:105  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
904:444  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
929:166  error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon
976:54   error  문장 끝에 쌍점(:)과 쌍반점(;)은 사용하지 않습니다  no-endline-colon

✖ 41 problems (41 errors, 0 warnings)

husky > pre-commit hook failed (add --no-verify to bypass)
```

**Comment by sukjae** [comment]
if i run `git commit ` without `yarn check-all`, doesn't cause any problem :(

```
➜  ko.reactjs.org git:(translate-uncontrolled-components) ✗ git add -A && git commit -m "Translated docs/uncontrolled-components page"
husky > pre-commit (node v8.11.1)
  ↓ Stashing changes... [skipped]
    → No partially staged files found...
  ✔ Running linters...
[translate-uncontrolled-components 91de4bb1] Translated docs/uncontrolled-components page
 1 file changed, 82 insertions(+), 84 deletions(-)
 rewrite content/docs/uncontrolled-components.md (69%)
```

**Comment by taehwanno** [comment]
Thanks for reporting this. this is caused by `yarn generate-ids` in `yarn check-all`. ids are generated by a script in a header that does not have an explicit id.

I submit the pull request for this issue at reactjs/reactjs.org#1905

**Comment by taehwanno** [comment]
Resolved by 066479b. I will close this.
